demos/datepicker/datepicker-fr.js

```diff
@@ -6,7 +6,7 @@
 	if ( typeof define === "function" && define.amd ) {
 
 		// AMD. Register as an anonymous module.
-		define([ "../jquery.ui.datepicker" ], factory );
+		define([ "datepicker" ], factory );
```

Comment:
I'd be happy to have some other solution that avoids this problem. I'm just not sure what we'd do other than parsing the actual dependencies and determining when we grab files that aren't `/ui/*` so that we know they won't be in the CDN build.